### ISSUE TITLE
Docker, add UID and GID env var

### ISSUE THREAD

**Issue by rgarrigue** [jpillora/cloud-torrent#259]
Hello

Can you please add a way to control the UID and GID for the downloaded file ? Something like this in the dockerfile

```
FROM whatever
ENV UID=1000
ENV GID=1000
RUN groupadd -g $GID cloudtorrent
RUN useradd -m -u $UID -g $GID -s /bin/bash cloudtorrent
USER cloudtorrent
CMD cloudtorrent
```

My use case is, I'm using cloudtorrent to download files in a torrent folder which is then shared / synchronized via NextCloud. NextCloud container use UID 33 / GID 33, actually cloudtorrent use root, hence UID 1 / GID 1. So NextCloud can't see downloaded torrent. 

The same problem will occur for any similar case.

**Comment by jpillora** [comment]
Yes this would be useful, same issue when integrating with a Plex container
